### PR TITLE
Improve formatting and update info in "method lookup" section

### PR DESCRIPTION
**Pull request by camelid**

### CODE REVIEW

File: src/method-lookup.md [hunk 1]
```diff
@@ -6,18 +6,11 @@ file provides an overview of the process. More detailed notes are in
...
-```
-
-into a more explicit [UFCS] form:
-
-```rust,ignore
-Trait::method(ADJ(receiver), ...) // for a trait call
-ReceiverType::method(ADJ(receiver), ...) // for an inherent method call
-```
+- `Trait::method(ADJ(receiver), ...)` for a trait call
+- `ReceiverType::method(ADJ(receiver), ...)` for an inherent method call
```
Comment (jyn514) [inline src/method-lookup.md:13]: Hmm, this doesn't seem any more clear to me? Why did you change it?

Comment (camelid) [reply src/method-lookup.md:13]: I think it makes it clearer that it's showing one of two options. The way it was before made it seem like it turned the one line `self.` form into two lines of an explicit form. Also, I think it looks better and is clearer when the comment is turned into proper Markdown text.

Here's a before and after, for comparison:

image

image

**Review by jyn514 — APPROVED** [review]
